src/Lykke.AlgoStore.Api/Infrastructure/ContentFilters/RequiredPermissionFilter.cs

```diff
@@ -26,19 +26,11 @@ public void OnActionExecuting(ActionExecutingContext context)
 
             // Check if the action needs a permission
             var reflectedMethod = context.Controller.GetType().GetMethods().Where(m => m.ReturnType == typeof(Task<IActionResult>) && m.Name == requiredPermission).FirstOrDefault();
```

Comment:
I think that @iliyan-georgiev-ph mentioned this before, but you can get reflected method like this:
var reflectedMethod = (context.ActionDescriptor as ControllerActionDescriptor)?.MethodInfo;